types/src/write_set.rs

```diff
@@ -280,6 +302,26 @@ impl WriteSetV0 {
     pub fn get(&self, key: &StateKey) -> Option<&WriteOp> {
         self.0.get(key)
     }
+
+    pub fn get_value(&self, key: &StateKey) -> Option<u128> {
```

Comment:
Hmm.. get_value function name is generic but it is returning `u128`, which is specific to total supply.. how about we change this to get_total_supply instead?

Reply:
ok done